.cci.jenkinsfile

```diff
@@ -35,6 +35,46 @@ coreos.pod([image: 'registry.fedoraproject.org/fedora:30', privileged: true, kvm
               cosa_cmd("buildupload --dry-run s3 --acl=public-read my-nonexistent-bucket/my/prefix")
           }
       }
+
+      stage("Pruning test") {
+          // Test that first build has been pruned
+          cosa_cmd("build ostree --force-image")
+          cosa_cmd("build ostree --force-image")
+          cosa_cmd("build ostree --force-image")
+          coreos.shwrap("cat /srv/builds/builds.json")
+          coreos.shwrap('jq -e ".builds|length == 3" /srv/builds/builds.json')
+          coreos.shwrap('jq -e ".builds[2].id | endswith(\\"0-1\\")" /srv/builds/builds.json')
+
+          // Test --skip-prune
+          cosa_cmd("build ostree --force-image --skip-prune")
+          coreos.shwrap("cat /srv/builds/builds.json")
+          coreos.shwrap('jq -e ".builds|length == 4" /srv/builds/builds.json')
+          coreos.shwrap('jq -e ".builds[3].id | endswith(\\"0-1\\")" /srv/builds/builds.json')
+
+          // Test prune --dry-run
+          cosa_cmd("prune --workdir /srv --dry-run")
+          coreos.shwrap("cat /srv/builds/builds.json")
+          coreos.shwrap('jq -e ".builds|length == 4" /srv/builds/builds.json')
+          coreos.shwrap('jq -e ".builds[3].id | endswith(\\"0-1\\")" /srv/builds/builds.json')
+
+          // Test --keep-last-n=0 skips pruning
+          cosa_cmd("prune --workdir /srv --keep-last-n=0")
+          coreos.shwrap("cat /srv/builds/builds.json")
+          coreos.shwrap('jq -e ".builds|length == 4" /srv/builds/builds.json')
+          coreos.shwrap('jq -e ".builds[3].id | endswith(\\"0-1\\")" /srv/builds/builds.json')
+
+          // Test prune --keep-last-n=1
+          cosa_cmd("prune --workdir /srv --keep-last-n=1")
+          coreos.shwrap("cat /srv/builds/builds.json")
+          coreos.shwrap('jq -e ".builds|length == 1" /srv/builds/builds.json')
+          coreos.shwrap('jq -e ".builds[0].id | endswith(\\"0-4\\")" /srv/builds/builds.json')
+
+          // Test prune --keep-last-days=5 - this has no effect currently
```

Comment:
Let's just drop this test if we can't actually verify the result. There's ways to hack it so that we can, but let's not block on that. This PR has been outstanding for long enough :)

Reply:
Done